src/components/Launch.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Image } from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+
+const Launch = () => {
+  return (
+    <Image source={require('../../images/fulllaunch.png')} style={styles.container} />
+  )
+}
+
+const styles = EStyleSheet.create({
+  container: {
+    flex: 1,
+    width: null,
+    height: null,
```

Comment:
This seems kind of odd. Is this the standard way to have an image take up the entire page rather than using 100%?